neural_speed/core/layers/ip_fusion_ffn.cpp

```diff
@@ -684,6 +684,11 @@ bool bestla_fusion_FFN_SiLu_f32f32_support(void* w1ptr, void* w2ptr, void* w3ptr
   return ffn_3w::bestla_fusion_ffn_f32f32_support(w1ptr, w2ptr, w3ptr, seq, fin, fmid, fout);
 }
 
+bool bestla_fusion_FFN_Gelu_Mul_f32f32_support(void* w1ptr, void* w2ptr, void* w3ptr, int seq, int fin, int fmid,
```

Comment:
Does it mean gemma FFN-8W matmul? or I misunderstanding? And what is the difference between `Gelu_Mul` and `Gelu`?

Reply:
FFN_Gelu_Mul is self.down_proj(self.act_fn(self.gate_proj(x)) * self.up_proj(x)),and FFN_Gelu is  self.fc_out(self.act(self.fc_in(x)))

Reply:
my bad, `3W` here means three weights ptr rather than three times dimension...